rtc/faq.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: default
+title: RTC frequently asked questions and troubleshooting
```

Comment:
We need to remove this ASAPP. Not in this PR because it's too late. FAQ pages are terrible for a range of reasons and should never be used. At a cursory glance, everything on this page would fit elsewhere in the docs and would more findable elsewhere.

These cover some of the reasons:
- https://thegood.com/insights/faq-pages/
- https://gds.blog.gov.uk/2013/07/25/faqs-why-we-dont-have-them/

Reply:
Could we Jira up an improvement on this please

Reply:
Not really agreeing here, FAQ can serve a good purpose. Just from own experience It certainly cut the support load on MoxieManager around some common setup issues. Also a good place to list "known issues" during Beta development.